CONTRIBUTING.md

```diff
@@ -18,20 +18,28 @@ Articles should cover any topic related to practical, day-to-day usage of these
 
 - Submit articles with a PR on the [theheadless.dev repo](https://github.com/checkly/theheadless.dev)
 - Write articles in markdown and can use the [Markdown extensions provided by Vue press](https://vuepress.vuejs.org/guide/markdown.html)
-- Add articles to the `posts` directory as markdown files 
-- Add code examples to the `snippet` directory as Javascript files and inserted into markdown code blocks using the `<< @snippet` syntax 
+- Add articles to the `posts` directory as markdown files
+- Add code examples to the `snippet` directory as Javascript files and inserted into markdown code blocks using the `<< @snippet` syntax
 - Code examples should cover both Puppeteer and Playwright where applicable
 - Add your full name and GitHub username in the relevant `author` and `githubUser` front matter
 
 You can find additional styling guidelines in this [dedicated file](https://github.com/checkly/theheadless.dev/blob/master/STYLE.MD). Our team will gladly help you apply these after the initial review.
 
+### How to contribute
```

Comment:
I like your change. Maybe we want to take it one step further and change naming and/or restructure the document? We now have:

CONTRIBUTING.md:
# Contributing a guide or article to theheadless.dev
## Contributing Content
### How to contribute

Weird structure, not very immediate.

Reply:
Sounds good. Can you propose something? @ragog

Reply:
@hlenke sounds good, I will follow up on this.